### PR TITLE
COM-3551: Add markdown lint

### PR DESCRIPTION
**Pull request by vkosinov**
COM-3551

## Checklist
- [ ] `data-testid` attribute has been set for new components and interactive blocks according to [documentation](https://github.com/CSSSR/e2e-tools/blob/master/docs/DATA_TESTID.md) 
- [x] PR is directed to the release branch (if there is a release branch, otherwise this item can be removed)
- [x] Code meets requirements not only from the description of the task, but also from its comments
- [x] Code works correctly on all resolutions
- [x] Task completed with comments for testing (if necessary, otherwise this item can be removed)
- [x] Browser dev tools has no errors
- [ ] Got acquainted with the project documentation and updated it if necessary

## Optional (if you have something to add, otherwise this heading and the paragraph below can be removed)
https://github.com/DavidAnson/markdownlint
https://github.com/igorshubovych/markdownlint-cli

### CODE REVIEW

File: _posts/ru/lint-your-css.md [hunk 1]
```diff
@@ -73,20 +73,20 @@ tag: 'web-development'
...
 ```stylus
 body
-	font-family PTSansBold
+ font-family PTSansBold
 .wrap
-	width 1000px
+ width 1000px
         margin 0 auto
-	background-image url(../images/background.png)
+ background-image url(../images/background.png)
```
Comment (ArtyomResh) [inline _posts/ru/lint-your-css.md:80]: Кажется, вот тут странные отступы линтер сделал

File: _posts/ru/ng-options.md [hunk 2]
```diff
@@ -213,10 +213,10 @@ $scope.setDefaultCage = function () {
 
 ```html
 <div class="form-group">
-	<select ng-model="request.volary" ng-change="setDefaultCage()" ng-options="volary.text for volary in volaries track by volary.volary_id" class="form-control"></select>
+ <select ng-model="request.volary" ng-change="setDefaultCage()" ng-options="volary.text for volary in volaries track by volary.volary_id" class="form-control"></select>
```
Comment (ArtyomResh) [inline _posts/ru/ng-options.md:216]: Мне кажется тут лучше настроить двойной пробел или два таба

File: _posts/ru/semantics-in-programming.md [hunk 3]
```diff
@@ -227,17 +221,16 @@ JSX должен быть в JS, а не JS в JSX. Top level синтакс
 </Note>
 
 ```jsx
-<Component />
+;<Component />
```
Comment (ArtyomResh) [inline _posts/ru/semantics-in-programming.md:224]: Кажется линтер лишнего добавил

File: _posts/ru/side-effects.md [hunk 4]
```diff
@@ -5,10 +5,11 @@ author: 'Андрей Мельников'
 date: '2017-10-07T00:00:00.000Z'
 tag: 'web-development'
 ---
+
 <Subtitle>
   > Начинать нужно с того, что сеет сомнение.
   >
-  > -- <cite>Братья Стругацкие</cite>
+  > -- Братья Стругацкие
```
Comment (ArtyomResh) [inline _posts/ru/side-effects.md:12]: Тут линтер тоже лишнего порезал

File: _posts/ru/side-effects.md [hunk 5]
```diff
@@ -371,15 +374,14 @@ Cкорее всего, мы добавим специальную опцию 
...
+- нет возможности хоть как-то локализовать проблему: <span class="no-wrap">что-то</span> не работает, а почему — непонятно.
 
 #### Cайд-эффекты непредсказуемы и не воспроизводимы
 
->Всё течёт, всё меняется, никто не может дважды войти в один и тот же поток, и к смертной сущности никто не прикоснется дважды!
+> Всё течёт, всё меняется, никто не может дважды войти в один и тот же поток, и к смертной сущности никто не прикоснется дважды!
 >
->-- <cite>Гераклит</cite>
-
+> -- Гераклит
```
Comment (ArtyomResh) [inline _posts/ru/side-effects.md:384]: тут тоже линтер перестарался)

File: _posts/ru/side-effects.md [hunk 6]
```diff
@@ -495,15 +497,15 @@ printToScreen(value: number): PrintedSumToScreenEffect
...
 
->Люди иногда спрашивают: «Что служит аналогом UML для Haskell?». Когда меня впервые спросили об этом 10 лет назад, я подумал: «Ума не приложу. Может быть, нам стоит придумать свой UML». Сейчас я думаю: «Это просто типы!». Люди рисуют UML-диаграммы, чтобы понять общую схему программы. Именно этим занимаются программисты на функциональных языках и на Haskell, когда придумывают сигнатуры типов для модулей и функций в этих модулях.
+> Люди иногда спрашивают: «Что служит аналогом UML для Haskell?». Когда меня впервые спросили об этом 10 лет назад, я подумал: «Ума не приложу. Может быть, нам стоит придумать свой UML». Сейчас я думаю: «Это просто типы!». Люди рисуют UML-диаграммы, чтобы понять общую схему программы. Именно этим занимаются программисты на функциональных языках и на Haskell, когда придумывают сигнатуры типов для модулей и функций в этих модулях.
 >
->-- <cite>[Саймон Пейтон Джонс](http://fprog.ru/2010/issue6/interview-simon-peyton-jones/)</cite>
+> -- [Саймон Пейтон Джонс](http://fprog.ru/2010/issue6/interview-simon-peyton-jones/)
```
Comment (ArtyomResh) [inline _posts/ru/side-effects.md:508]: И тут

**Review by ArtyomResh — CHANGES_REQUESTED** [review]

**Review by ArtyomResh — APPROVED** [review]

**Comment by vkosinov** [comment]
The expiration date has passed

**Pull request closed without merging**